### DISCUSSION TITLE
Current issues with osu!lazer mod multipliers

### DISCUSSION THREAD

**Discussion by osuKingling** [ppy/osu#29486] [Ideas]
With the recent addition of the daily challenge along with personally completely transitioning to lazer, I believe there are a quite a few flaws with the current mod multipliers. After speaking with other players I don't seem to be alone in this opinion so i'm writing this up to hopefully bring more attention to the current issues, along with a few proposals to fix them.

### **Issue 1: DT Multiplier**

With the addition of rate change, it has become clear the multiplier between the intervals is currently too small. This is causing higher rate plays to require too high of a relative accuracy to beat a lower rate. An example to show this is the leaderboard of the [daily challenge from July 26th](https://osu.ppy.sh/beatmapsets/1934595#osu/3997798).

![image](https://github.com/user-attachments/assets/96d16973-0956-489d-90d0-feb43d2344a4)

In the image above you can see the 99.33% 1.80x play is above both the 2.0x and 1.9x plays. It seems be common consensus that the 2.0x score shown is much harder to achieve than the 1.80x play however the multipliers don't reflect this.

Another issue with the current DT multiplier is it's interaction with HR. Currently a HRDT(1.1x) score is weighted higher than a DT(1.4x) score. This one is slightly more map dependent, however from experience playing the daily challenges speeding up HR is often much easier than the DT only equivalent speed. Below is an example from the [daily challenge from August 5th](https://osu.ppy.sh/beatmapsets/2095154#osu/4423300).

![image](https://github.com/user-attachments/assets/d6ed9caf-be13-4b75-96a9-b922a42666a6)

As you can see, my HDHRDT(1.1x) score with 99.27% beats the HDDT(1.5x) score, despite HDHRDT(1.1x) being only 5.82* yet regular HDDT is 7.5*. Of course we all know star rating is not 100% accurate, however this is a huge discrepancy. 

**Proposal to fix: increase the current multiplier from +0.02x per 0.1 speed, to either +0.04x or +0.05x per 0.1** 

### **Issue 2: HD/HR Multipliers**

It's been asked for a long time in the community, but due to the new changes in score system weighting accuracy more it has become even more important to give HR a higher multiplier than HD. On almost all maps, HR is significantly harder than HD due to the fact it increases the mechanical difficulty of a map whereas HD only changes the reading difficulty. HR also increases the OD making it harder to get high accuracy, which is why in the new score system it is even harder to beat HD scores with HR. 

There is a number of ways to fix this, however I personally believe the best solution is to both nerf the HD multiplier, and slightly buff the HR multiplier. This is because in tandem with the DT changes above, HR would need a slight increase in order to keep it's weighting relative to that in check. 

**Proposal to fix: Decrease the HD multiplier from 1.06x to 1.02x, increase the HR multiplier from 1.06x to 1.10x**

### **Issue 3: Classic mod multiplier**

This is probably the most controversial issue, however it seems to be universally agreed that the current classic mod is too low. In most cases the difficulty introduced by slider accuracy is not enough to outweigh the current 0.96x multiplier, so I believe an increase is required.

Further more, currently mod combinations like HDHR (1.13x) outweigh CLHDDT (1.12x) which is obviously unfair. 

Quite a few people would like it to be 1.0x in order to preserve the integrity of old scores however I personally disagree with that as playing with slider acc is just objectively harder, and also the correct direction for the game. This debate is for another post though, so I will assume we are keeping it less than 1.0x.

**Proposal to fix: increase classic mod multiplier from 0.96x to 0.98x** 

### **Footnote**

Obviously these changes need a formal discussion, so I encourage people to give their input in the comments on my proposals, aswell as bringing up any other issues I may have missed.

As a sidenote: a lot of these issues are also worsened by the new score system, and whilst I believe the new score system is a huge improvement to old scorev1 I do think a few tweaks there would also help with these (primarily slightly decreasing the weight of accuracy). That is out of scope of this post though so may write up another one of these for that.

**Comment by Exploity** [comment]
Another easily noticeable multiplier issue currently is HDHRFL being worth more than HDDTHR even tho the latter is way harder on most maps.

**Comment by Givikap120** [comment]
1.06 for HD is fine
It's only too big when HD is together with DT
1.02 will make NM almost always superior to HD

HD should be 1.05
HR should be 1.1
and DT should be like 1.3
HR is similar difficulty to DT 1.2x for average 5-6 star map (and this star range should be main focus as most active players are playing here)

DT multipliuer = 1.3 means that:
DT 1.1x -> 1.06
DT 1.2x -> 1.12
DT 1.3x -> 1.18
DT 1.4x -> 1.24
DT 1.5x -> 1.3
DT 1.6x -> 1.36
DT 1.7x -> 1.42
DT 1.8x -> 1.48
DT 1.9x -> 1.54
DT 2.0x -> 1.6

FL and BL should probably be around 1.2

Those multipliers should be tested tho

**Reply by osuKingling** [reply]
Just one thing to reply to this

> 1.02 will make NM almost always superior to HD

What do you mean by this? For the majority of maps HD doesnt make the map harder to acc, and for most people its entirely a preference mod. Lots of people even prefer it on most maps. I don't think its unreasonable to think that it should be very close to NM and you would still require ~+0.5% accuracy to beat it with nomod with a 1.02x multiplier.

**Reply by Givikap120** [reply]
It's not a preference mod when it makes the map harder
And HD definitely make maps harder to acc by the very simple fact of having no indication of timing
In some BotD map I increased my acc from 91% to 95% by disabling HD and this is considering that I have a lot of HDDT in my top plays
A lil bit too much of a difference for "doesn't make the map harder to acc"
Really reminds me of "acshually AR8 is a preference" people when AR8 makes map marginally harder to aim and to acc because of 2x density

**Reply by osuKingling** [reply]
I think having a sample size of just yourself isn't the best way to view it, I know of many other players who either find no difference with HD, or even find HD easier than nomod. I'm welcome for others to give their opinions too but I dont think using just yourself is a fair argument.

Also FWIW, you do still have an 'indication of timing' with HD. It's just not an approach circle, its using the timing of the fade. You can argue it might not be as good of an indicator but experienced HD players can definitely use the timing of the fade to assist with timing. 

(I'm also not saying theres never scenarios where HD would be widely considered as harder than nomod, but I think it makes as little enough of a difference to enough people on enough maps to warrant a low multiplier.)

**Reply by Givikap120** [reply]
The best way to view it is the fact that even players like mrekk, who played HDDT for eternity - turning off HD on some maps, and those maps have 10.5+ AR
And this is considering brokenly high HD reward for High AR, on AR10 mrekk and most top players play HD like never
Just this fact alone is insanely clear sign that HD always make maps harder and in many case marginally, unless you're absurdly skill issued on NM (the same as people who say that 200bpm streams are easier than 170bpm or that HR is easier than NM)

**Reply by cl8n** [reply]
as an HD player 1.02x sounds good and kingling's reasoning tracks with my own experience (even 1x I would not mind but I'm assuming that's unpopular). I always felt like this mod gives me undeserved score for what is largely just my preference. 1.02x seems fair to me as a small compensation for the maps where HD is widely considered to be more difficult, though I don't personally feel it

givikap it would help in these discussions if you don't make unreasonable arguments, like claiming something is an "insanely clear sign that HD always make maps harder" based on a small sample of player behavior...

to be fair in defense of HD's current multiplier though, it's hard to tell in advance what the difference of 1.02x and 1.06x (etc) would actually look like in practice, if all of the other important mod multipliers were increased as per this proposal. HD is the smallest remaining multiplier either way so it may not make a big difference, I don't know

**Reply by Givikap120** [reply]
> as an HD player 1.02x sounds good and kingling's reasoning tracks with my own experience (even 1x I would not mind but I'm assuming that's unpopular). I always felt like this mod gives me undeserved score for what is largely just my preference. 1.02x seems fair to me as a small compensation for the maps where HD is widely considered to be more difficult, though I don't personally feel it
> 
> givikap it would help in these discussions if you don't make unreasonable arguments, like claiming something is an "insanely clear sign that HD always make maps harder" based on a small sample of player behavior...
> 
> to be fair in defense of HD's current multiplier though, it's hard to tell in advance what the difference of 1.02x and 1.06x (etc) would actually look like in practice, if all of the other important mod multipliers were increased as per this proposal. HD is the smallest remaining multiplier either way so it may not make a big difference, I don't know

Accuracy difference for 1.05 multiplier is enough to compensate 97.4% acc HD score compared to 99% NM score.
Converted to stable multiplier - it's the same as multiplier 1.016 for stable scoring.
1.05 multiplier is similar to multiplier for HD in pp for AR10 8 star map, as pp have enough bonus to compensate 97.7% HD score to be worth the same as 99% NM score (the exact multiplier to match pp here is 1.04).

HD players opinion is much less relevant because as they played only HD - they're most likely highly skill issued on NM. The fact that most maps without HR/DT FC have NM FC and don't have HD FC tells a lot about HD being objectively harder and it's NOT a preferential mod (at least not in full sense). This is not small sample of player behaviour, actually the people who consider HD the same difficulty as NM are very very small sample of players. Otherwise - we would've seen >99% players playing HD in all cases, what is not even close to truth, as HD is only widely used in pair with DT to get overweight HD pp bonus on high AR.

**Reply by osuKingling** [reply]
@Givikap120 You're still looking at this from a very bias POV in my opinion. 

Your main bias is not accounting for the fact the majority of players *only* play nomod for a large part of their early playing time (since its the default obviously, and it's not like brand new players care about PP or score or anything), so of course they will naturally find hidden harder unless they attempt to adjust and learn it, and will result in fewer HD specialists. It's also hard for brand new players to jump in straight with HD due to the low AR of lower difficulties, so it's a skill that has to be adjusted to later on. 

Once players start playing 4*+, the overwhelming majority of maps are AR9+ so the low AR issue becomes much less relevant and these are the maps where the HD multiplier will have most impact for leaderboards.

Secondly, saying:

> HD players opinion is much less relevant because as they played only HD - they're most likely highly skill issued on NM

Can just be reversed for the exact same thing. NM players opinions are irrelevant because they're skill issued on HD. Doesn't really make any sense. 

And lastly, I don't even believe that the statement of 'The fact that most maps without HR/DT FC have NM FC and don't have HD FC' is true. I had a brief look through newly ranked maps with either no HR/DT FC or no FC at all and there was still plenty of HD scores in the top ranks. 

Here are the first ones I found when filtering between 6* - 7.5* with no (or few) HR/DT FCs ranked recently:

https://osu.ppy.sh/beatmapsets/613347#osu/1294084
https://osu.ppy.sh/beatmapsets/2220009#osu/4705868
https://osu.ppy.sh/beatmapsets/2204750#osu/4667149
https://osu.ppy.sh/beatmapsets/2075254#osu/4344423
https://osu.ppy.sh/beatmapsets/2151228#osu/4532438
https://osu.ppy.sh/beatmapsets/1753391#osu/3702674
https://osu.ppy.sh/beatmapsets/1711172#osu/4636292

The ratio of HD scores seems very reasonable to me. (Also feel free to look at it yourself, these are quite literally the first ones that popped up in osuweb beatmap search with stars>6  stars<7.5 filters on, so you know i'm not cherry picking)

**Reply by cl8n** [reply]
givikap I tried to write a reply to this but it became less about our differing opinions about HD's affect on score and more just picking apart why argumentatively I think what you're saying is flawed. I don't think it makes sense to treat this as an attempt to derive HD's multiplier from some kind of objective measure in the first place -- it's all up to our preferences and experience playing HD vs. NM, and how we feel that should factor into score. I'm just much closer to the "1x" side of this argument because I personally feel little to no difference in difficulty between HD and NM. also, I shouldn't have to clarify this, but I'm not your convenient strawman that somehow got worse at NM by maining HD... again I think it's not helpful to the discussion to be including stuff like that in your posts

thanks for the scoring examples though. 1.05x has more of an effect in your example than I'd personally like to see but it's not as bad as I thought since lazer's acc scaling effectively takes some power away from every mod multiplier at the top end. I guess I don't have a big issue with 1.05x if the more popular opinion ends up being to "keep HD around where it already is" or something

**Reply by Givikap120** [reply]
> Your main bias is not accounting for the fact the majority of players only play nomod for a large part of their early playing time
You know what also majority of players play in their early playing time? TV size jumps. Is this relevant? No, because early playing time doesn't decide what you will play. 

Many players are immediately jumping on HDDT instead of DT-only as they learn that they can get noticeable pp bonus for minor difficulty increase. 

> Can just be reversed for the exact same thing. NM players opinions are irrelevant because they're skill issued on HD. Doesn't really make any sense.

No, it couldn't be reversed because very big amount of NM players are playing DT with HD. The players who play both NM and DT without HD opinion is irrelevant, yes. 

> so you know i'm not cherry picking

No, you're indeed cherry picking by picking the 6 maps for no reason. Those maps LB scores is literally the matter of random top player playing it with HDHR.
There's 5 Jashin FCs and none of them are HD, why are they don't claim "free" pp? And you can't say that it's because they're NM players.
- ASecretBox - have dozens of HDHR scores in top100
- Rafis - literally played HDDT for eternity, most likely have most of his hours played with HD
- VoProSSoFF - have a lot of HDDT scores
If HD haven't made this map significantly harder - they would've claimed this free HD pp, but they haven't. Why? Do you have explanation for this?

Okay, flow aim map, not fair. Crazybanger - 3 NM FCs, 0 HD FCs. mrekk, Plasma and Forum all have very big experience with HD, why not claim free #1 HD FC??? And look just on HD scores in general - in top50 there are only 2 HD scores. And 4 HR scores lol. 0 difficulty increase you say?
Maybe map nitpicking? Check every other 9-10 star aim map. Almost every score in top50 is NM, and coincidentally those who are setting NM scores usually have full top100 of HDDT.

**Reply by Givikap120** [reply]
> givikap I tried to write a reply to this but it became less about our differing opinions about HD's affect on score and more just picking apart why argumentatively I think what you're saying is flawed. I don't think it makes sense to treat this as an attempt to derive HD's multiplier from some kind of objective measure in the first place -- it's all up to our preferences and experience playing HD vs. NM, and how we feel that should factor into score. I'm just much closer to the "1x" side of this argument because I personally feel little to no difference in difficulty between HD and NM. also, I shouldn't have to clarify this, but I'm not your convenient strawman that somehow got worse at NM by maining HD... again I think it's not helpful to the discussion to be including stuff like that in your posts
> 
> thanks for the scoring examples though. 1.05x has more of an effect in your example than I'd personally like to see but it's not as bad as I thought since lazer's acc scaling effectively takes some power away from every mod multiplier at the top end. I guess I don't have a big issue with 1.05x if the more popular opinion ends up being to "keep HD around where it already is" or something

If there was no difference in difficulty in HD compared to NM - we would've seen everyone playing with HD
What is exactly what happens on HDDTHR, because on AR11 there's indeed little to no difference. But moving to AR10 - and the difference is becoming very clear and noticeable

**Reply by cl8n** [reply]
I can't tell if it's by malice or ignorance but your first words there ignore my entire point and attempt to refute something I didn't even say. (that's probably my last reply here because I'm just getting annoyed now and don't want to bring that vibe to github )

**Reply by Givikap120** [reply]
> something I didn't even say

> I'm just much closer to the "1x" side of this argument because I personally feel little to no difference in difficulty between HD and NM

The fact that top players play with HD almost always when it comes to AR11 and much more rarely when it comes to AR10 - is enough evidence to understand that calling HD a "preferential mod" is fundamentally wrong. It's not about the preference - it's about the HD adding more and more difficulty as AR goes lower and lower.

It's the same situation as some people calling low AR "a preference" when in reality it's just as difficulty increasing as every other way to increase the difficulty such as CS increase or Rate increase. The only difference is that it works in much less predictable and simple way.

If some people think that 200bpm stream is easier than same stream but slowed down to 170bpm - it's not a preference - it's a skill issue on 170bpm.

You can call it "malice" or "ignorance" as much as you want, but was you who ignored the fact of top players playing HD 24/7 when it comes to AR10.6 and not playing HD when it comes to AR10 and especially AR9, and then based your "preference" point on this fact not existing.

**Reply by cl8n** [reply]
you keep trying to tell me evidence for why you think HD has an objective affect on difficulty and I'm saying that I don't believe this approach is appropriate for deciding a score multiplier for HD. my opinion on HD's score multiplier comes from my personal experience and my desire to see it not affect leaderboards so much as a result. it's not that deep. I "based [my] 'preference' point" on the fact that I have preferences... to which your whole top player behavioral study thing is obviously unrelated; I haven't ignored that at all, I directly addressed how I felt about "[deriving] HD's multiplier from some kind of objective measure"

(okay I actually unsub this time because I forgot to do that :cry: (please stop making threads miserable) (also I clearly failed at not bringing annoyedperson energy to github but u dont win em all i guess))

**Reply by osuKingling** [reply]
This will also be my final reply but clayton summed things up well

Whether or not you want to go study every top player ever or try to put objective measures on things, the reality is there is a not so small subset of players who will tell you directly that they prefer HD to nomod and find it easier or the same. 

Look at maliszewski, cookiezi, xootynator, woey, etc. All these guys clearly only play with HD, set all their best scores with it and show no signs in their plays that nomod is sizably easier for them on lower ARs. What do you say about those players? Do you think if they took HD off they would be able to easily set better scores?

tl;dr you keep trying to put this objective analysis on everything whilst also dismissing any player who says they prefer HD as 'skill issued on nomod'. I just think your argument is biased and disingenuous because you are quite literally dismissing a player (clayton) who is telling you they prefer HD and calling them skill issued.

**Reply by Newbie1st** [reply]
As a HD player I'd argue that 1.05x for HD and 1.15x for HR would be a better fit for lazer's current acc scaling on standardized scores, the HD being objectively harder than HR on low SRs is irrelevant, since most of the leaderboards at that level get filled with 3 or 4 mod SSes almost instantly, and that stops being the case roughly at the same point where HR overtakes HD in difficulty. For DT I was thinking of 1.25x to make things neater, but 1.3x would be a better way to prevent HDFL overtaking DT only scores. DT and HR multiplier being bigger will also make good runs with those appear higher on the leaderboards on high end maps, since low misscount HR scores are usually much more impressive than FCs from 7* onwards.

**Comment by felipemarins** [comment]
I would like to add that the Traceable mod have a 1x multiplier currently, which doesn't make much sense IMO. I would suggest it to change to 1.06x like Hidden or something close.

**Reply by osuKingling** [reply]
yup i agree, whenever the skin exploit with that mod is fixed I think it should be the same as HD

**Comment by Moltanicaa** [comment]
Imo - HR should be obsoleted when MR will ranked

**Comment by cihe13375** [comment]
I think we can make a simple comparison between scorev1 multipliers and lazer multipliers

Let's say a NM SS play has the same score with an HR FC with X% accuracy.
In scorev1 it roughly means X%*1.06=1, so X=94.3%
Assuming that we want to keep this value X same on lazer. On lazer an FC with 94.3% accuracy has a score of (if I recall the formulas correctly...)
500k + 500k * (94.3%)^5 = 873k
it should be same as a NM SS (1000k) after the score multiplier is applied, so the multiplier is 1000k/873k = 1.145

The same calculation can be done for DT, giving a multiplier of 1.275 (~784k before the multiplier)

The calculations above did not consider non-FC plays, because in scorev1 it's very difficult to beat FC plays with non-FC unless it misses at the very beginning or end. In lazer this becomes possible, but I think it is an improvement. Also from my observation of LGA one need to perform quite well (reasonably close to FC) to get a ~800k score, so the combo part should not have a drastic effect.

The difficulty decreasing mods are different stories though, because in these cases we shall compare e.g. an EZ SS with a NM of X misses and Y% accuracy. That's probably not important if we only consider the top of leaderboards, though.

**Reply by osuKingling** [reply]
I don't think we want to convert scorev1 multipliers to lazer, in most cases they felt far too low imo. I think something like 96.5-97% is what we want for HR and ~99% for HD as opposed to what it is in stable. Thats my opinion at least, it would need more input for final values

**Reply by cihe13375** [reply]
maybe we need a community survey (probably within lazer players, since lazer acc is not very comparable with stable) with questions like "NM SS should be same as _____% FC HR" and derive the multiplier from the results (e.g. 97%~1.076x and 99%~1.025x). I recall that the lazer scoring formula itself was built in a similar way (asking players many questions like this and adjust combo/acc portion and exponent to fit the results).

**Comment by ziv-vy** [comment]
This post only looks at competitive osu!. It ignores lower star ratings. For casual play, I can safely say that **HD is harder than HR.** This change would completely nuke casuals. So we now have a casual issue
Keep things like how they are and pros complain
Change things and casuals complain.
The whole score multiplier system is flawed.
We should have a dynamic system where score is variably affected based on beatmap but that will complicate things even more, and we all love to know that HDHRFLDT(x2) has a multiplier of 1.52.

**Reply by osuKingling** [reply]
Yes, it is intended to only look at competitive osu. Why? Score multipliers are pretty much only relevant for *leaderboards*. The proposals in this post tries to take into account the most common mod combinations for all star ratings, e.g. HDDTHRFL on low stars, HDDT on midrange maps, HDHR on harder stuff etc. It definitely does **not** ignore lower star ratings, however it puts priority on the most important mod combinations for leaderboards.

Surely you can agree it is more important for the top 50 of leaderboards to be accurate over the average player's local scores?

(Sidenote, this is also under the premise higher PP plays will be submitted over higher score plays in due time.)

Also, with regards to:

> We should have a dynamic system where score is variably affected based on beatmap

This is the intention of PP, score is intended to be a oversimplified version that is easy to understand which of course will lead to some inaccuracies. I don't disagree with the notion that HD on lower star maps is harder than HR on average, but ultimately comparing HD only and HR only scores is a rare scenario as these scores are rarely high on a leaderboard so a sacrifice has to be made there.

**Reply by ziv-vy** [reply]
I definetly agree, but there's one more place where score actually mattets: Multiplayer lobbies.

**Reply by osuKingling** [reply]
Sure? If you open the lazer multiplayer lobby tab right now the average star rating is around 4-4.5ish, HR starts to become harder than HD (on average) usually at around 3.5* in my experience so this is another argument in favour of increasing the HR multiplier. 

(The reason its about 3-3.5* is because this is where AR starts hitting 10 and OD is usually above 9)

**Reply by osuKingling** [reply]
Also one last thing - the average player progresses from 1-2* to above 3* within the span of a few days to weeks, there is not many players who regularly/exclusively play maps where HD is harder than HR.

**Reply by ziv-vy** [reply]
Heavily varies tho. Sometimes it feels like all lobbies are 2* and other times it feels like all is 7*
Also I don't really agree with your 3.5* figure, but that's a controversial claim so I won't argue.

**Reply by osuKingling** [reply]
Worst case scenario the breaking point definitely isn't above 4*, and I don't think theres any particular weighting above or below this for the average multiplayer lobbies difficulty so I don't think your point above adds any additional weight to your idea

**Reply by Givikap120** [reply]
> This post only looks at competitive osu!. It ignores lower star ratings. For casual play, I can safely say that **HD is harder than HR.** This change would completely nuke casuals. So we now have a casual issue Keep things like how they are and pros complain Change things and casuals complain. The whole score multiplier system is flawed. We should have a dynamic system where score is variably affected based on beatmap but that will complicate things even more, and we all love to know that HDHRFLDT(x2) has a multiplier of 1.52.

having mrekk DT FC as map #1, surpassing HDHR FCs, is infinitely more important than 7 digit local scores that don't even show up on the leaderboards
and new player usually don't even play HD in the first place

**Comment by MrAntonym** [comment]
Another mod multiplier issue which I've discovered by playing the daily challenge is to do with half time (HT) vs. wind down (WD)/wind up (WU). The mod multiplier of HT scales based on how it is customized (from 0.1 to 0.5 as you go from 0.5x speed to 0.99x speed) which I think makes sense since there may be maps where even small slow-downs make the map much easier (such that 0.9 score for 0.9x speed or something similar might be viable for getting the top score on a leaderboard).

The mod multipliers for WD and WU work very differently. Regardless of what the initial and final speeds are, the mod multiplier is always 0.5. This means that in terms of your score, it is always strictly better to play WD/WU going from speed (a)x to speed (a-0.01)x (or vice versa) instead of playing HT at (a)x speed. Also, you can play at whatever speed from 0.5x to 0.99x is easiest for you while maintaining exactly the same mod multiplier.

My suggested change would be to make the WD/WU mod multiplier from (a)x speed to (b)x speed just be equal to the HT multiplier for (min(a,b))x speed, but I could plausibly see some bonus weight toward having a large value of |a-b|.

The example below is from today's (Oct 28-29) daily challenge where both myself and someone else have found ourselves in a sea of better players on the leaderboard thanks to being able to use WD in this way. Using WD/WU like this is still unranked, so it doesn't affect the regular leaderboards, but it seems like something that should be fixed even if only for the sake of the daily challenge.

![Screenshot 2024-10-28 at 8 51 20 PM](https://github.com/user-attachments/assets/2daa0a7d-fd4d-420e-b053-51b9275ca1cc)

**Comment by Serizawa-Haruki** [comment]
I'd like to revive the discussion regarding this issue since it's a very important aspect of competitive play and the current score formulas are extremely unbalanced.
A good example for this is the fact that HDDT scores can be beaten by HDHR scores even if the DT score has the same or better accuracy, for example https://osu.ppy.sh/beatmapsets/853600#osu/1836593
Even classic HDHR scores can overtake HDDT scores, such as in this example: https://osu.ppy.sh/beatmapsets/1092298#osu/2288017
